main.ml

```diff
@@ -420,6 +420,7 @@ let process root_dir name =
       "SR.detach",       "SR_DETACH";
       "SR.ls",           "SR_SCAN";
       "SR.stat",         "SR_UPDATE";
+      "SR.probe",        "SR_PROBE";
```

Comment:
GFS2 supports this but PVSproxy does not. This is fine because the above loop checks for the presence of the script before reporting the capability.